src/transform/CollectionUtilities.js

```diff
@@ -0,0 +1,76 @@
+import Polyfill from './Polyfill'
+
+/**
+ * Extracts array of page issues from element
+ * @param {!Document} document
+ * @param {?Element} element
+ * @return {!Array.<string>} Return empty array if nothing is extracted
+ */
+const collectPageIssues = (document, element) => {
+  if (!element) {
+    return []
+  }
+  const tables =
+    Polyfill.querySelectorAll(element, 'table.ambox:not(.ambox-multiple_issues):not(.ambox-notice)')
+  // Get the tables into a fragment so we can remove some elements without triggering a layout
+  const fragment = document.createDocumentFragment()
+  const appendTableToFragment =
+    table => fragment.appendChild(table.cloneNode(true)) // eslint-disable-line require-jsdoc
```

Comment:
Are we wanting to show page issues in multiple places? Just curious why this isn't a transform (_remove_ the tables from the passed in document and operate on them directly instead of cloning them).

Reply:
These utilities for the gathering part. I think there's an upcoming meeting ( @coreyfloyd ) to further iron out presentation of things like article footer menu across iOS, Android & mobile web... (on iOS for example, `Page issues` only appear in the article footer menu, but `Disambiguation` appears both inline and in the menu, which I think differs a bit from Android and mobile web)

Reply:
Oh regarding cloning into the fragment, the frag is just a convenient "scratchpad" we can use to easily remove elements we don't want represented when doing `innerHTML` or `textContent` (without removing them from the actual live document).

Reply:
I renamed the cloning `const` to be more declarative.